src/Sylius/Component/Locale/Provider/LocaleProviderInterface.php

```diff
@@ -20,5 +20,7 @@ interface LocaleProviderInterface
      */
     public function getAvailableLocalesCodes(): array;
 
+    public function isLocaleCodeAvailable(string $locale): bool;
```

Comment:
🚨 BC break 🚨

We need to create a new interface that may be implemented by the same class that we already have.

Reply:
Yeah, thanks for spotting it. I will add it to the description.

Reply:
ping @mamazu :)

Reply:
Well, the problem is that there is no good backwards compatibility fix. So I guess this will not be merged?

Reply:
Maybe add a new abstraction instead?

Reply:
Yeah, that's probably the best idea. I will do that.